### PR TITLE
rewrite docs

### PR DESCRIPTION
**Pull request by nslater**

### CODE REVIEW

File: docs/connect.rst [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+
+    crate:<HOST_ADDR_1>:<PORT>,<HOST_ADDR_2>:<PORT>
+
+Here, ``<HOST_ADDR_1>`` and ``<HOST_ADDR_2>`` are the host strings for the
+first and second CrateDB nodes, respectively.
+
+In fact, you can specify as many nodes as you like. Each corresponding host
+string must be separated from the previous one using a ``,`` character.
+
+The driver will attempt to connect to each node in the order they appear. The
```
Comment (seut) [inline docs/connect.rst]: that's not correct, one hosts is chosen by random (https://github.com/crate/crate-pdo/blob/master/src/Crate/PDO/Http/ServerPool.php#L79) to not always hit the same node.

File: docs/connect.rst [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+
+   If you are using CrateDB 2.1.x or later, you must supply a username. If you
+   are using earlier versions of CrateDB, this argument is not supported.
+
+   See the :ref:`compatibility notes <cratedb-versions>` for more information.
+
+.. TIP::
+
+   If you have not configured a custom `database user`_, you probably want to
+   authenticate as the CrateDB superuser, which is ``crate``. By default, the
```
Comment (seut) [inline docs/connect.rst]: It's not possible to define a password for the `crate` superuser, so it is not by default.

File: docs/connect.rst [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+    crate:<HOST_ADDR>:<PORT>/<SCHEMA>
+
+Here, replace ``<SCHEMA>`` with the name of the schema you want to select.
+
+.. TIP::
+
+   The default CrateDB schema is ``doc``, and if you do not specify a schema,
+   this is what will be used.
+
+   However, you can query any schema you like at query time.
```
Comment (seut) [inline docs/connect.rst]: I'd also add here "... by specifying it in the query."

File: docs/getting-started.rst [hunk 4]
```diff
@@ -1,76 +1,68 @@
...
-   sh$ composer require crate/crate-pdo:~0.7.0
+Add the driver package to you project `composer.json`_ file, like this:
 
-Then run ``composer install`` or ``composer update``.
+.. code-block:: json
 
-Inside your PHP script you will need to require the autoload file:
+    {
+      "require": {
+        "crate/crate-pdo":"~0.7.0"
```
Comment (seut) [inline docs/getting-started.rst]: latest release is 1.0.0

File: docs/types.rst [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+
+Query Values
+------------
+
+When using values in a query, you must bind them to a ``PDO`` `parameter class
+constant`_, using either `bindParam()`_ or `bindValue()`_.
+
+We recommend the following bindings:
+
+.. _bindParam(): http://de2.php.net/manual/en/pdostatement.bindparam.php
```
Comment (seut) [inline docs/types.rst]: I prefer all links at the bottom of a document

**Comment by nslater** [comment]
@seut pushed fixups! thanks!

File: docs/connect.rst [hunk 6]
```diff
@@ -0,0 +1,251 @@
...
+Here, ``<HOST_ADDR_1>`` and ``<HOST_ADDR_2>`` are the host strings for the
+first and second CrateDB nodes, respectively.
+
+In fact, you can specify as many nodes as you like. Each corresponding host
+string must be separated from the previous one using a ``,`` character.
+
+.. NOTE::
+
+   The driver will attempt to connect to each node in a random order appear.
+   The first successul connection will be used, and all other nodes will be
```
Comment (seut) [inline docs/connect.rst]: This is still not correct, I'm sorry I wasn't clear enough. On first connection the list of servers is shuffled to not always use the first declared server on connection creation. The list of servers is saved and on each statement execution the next available server is used and moved to the end of the list. So its a basic client side round-robin implementation. See https://github.com/crate/crate-pdo/blob/master/src/Crate/PDO/Http/ServerPool.php#L93

**Comment by nslater** [comment]
@seut ty! pushed a fixup!

**Pull request merged**